ines/ines.js

```diff
@@ -0,0 +1,21 @@
+let apples = 20;
+let oranges = 30;
+
+console.log("Line 4 is:", apples === oranges ? true : false);
+console.log("Line 5 is:", apples !== oranges ? true : false);
+console.log("Line 6 is:", apples < oranges ? true : false);
+console.log("Line 7 is:", oranges > apples ? true : false);
+
+let mangoes = 5;
+
+console.log("Line 11 is:", mangoes * apples > mangoes + oranges ? true : false);
+console.log("Line 12 is: ", (mangoes === oranges) === apples ? true : false);
```

Comment:
console.log(apples == oranges == mangoes);

Reply:
and no ternary?

Reply:
why only ==? and not ===?